src/bin/sage-runtests

```diff
@@ -52,6 +52,10 @@ if __name__ == "__main__":
              'if set to "all", then all tests will be run; '
              'use "!FEATURE" to disable tests marked "# optional - FEATURE". '
              'Note that "!" needs to be quoted or escaped in the shell.')
+    parser.add_argument("--hide", metavar="FEATURES", default="",
+        help='run tests pretending that the software listed in FEATURES (separated by commas) is not installed; '
+             'if "all" is listed, will also hide features corresponding to all non standard packages; '
```

Comment:
I don't know what "non standard" means here; do you mean "optional or experimental"?

Reply:
> do you mean "optional or experimental"?

Yes! I changed this, to have it more clear!